Makefile

```diff
@@ -94,11 +96,17 @@ clean: docs-clean
 
 BINS_ALL_TARGETS := zrepl-bin platformtest-bin vet lint
 GO_SUPPORTS_ILLUMOS := $(shell $(GO) version | gawk -F '.' '/^go version /{split($$0, comps, " "); split(comps[3], v, "."); if (v[1] == "go1" && v[2] >= 13) { print "illumos"; } else { print "noillumos"; }}')
+ifeq ($(GOARM), 7)
+	PLATFORM := $(GOOS)-$(GOARCH)hf
```

Comment:
I'd rather encode the `$GOARM` value as `$(GOARCH)v$(GOARM)`.
Also, please rename `PLATFORM` to `ZREPL_TARGET_TUPLE` to avoid confusion with the platform tests, and move the definition to the top of the file (i.e. right before `printvars`).

Reply:
I'm not shure what the right arch name is `armhf`, `armv7`, `armv7hl`:
Debian: apache2_2.4.38-3+deb10u3_armhf.deb
Ubuntu: apache2_2.4.41-1ubuntu1_armhf.deb
Fedora: httpd-2.4.41-13.fc32.armv7hl.rpm
FreeBSD: ports branch armv7
OpenSUSE: armv7hl
Arch Linux ARM: apache-2.4.41-1-armv7h.pkg.tar.xz

Reply:
used `$(GOARCH)v$(GOARM)` for now